src/sparseml/pytorch/utils/sparsification_info/module_sparsification_info.py

```diff
@@ -58,5 +59,13 @@ def from_module(cls, module: torch.nn.Module) -> "ModuleSparsificationInfo":
             quantization_info=SparsificationQuantization.from_module(module),
         )
 
-    def loggable_items(self) -> Iterable[Tuple[str, float]]:
-        raise NotImplementedError()
+    def loggable_items(self) -> Generator[Tuple[str, Any], None, None]:
```

Comment:
Thoughts on making it an `__iter__`, such that we can do:

```python
info = ModuleSparsificationInfo.from_module(...)
for tag, item in info:
    ....
```

Reply:
let's not do that for now since the logged items here should directly match with the logging targets from our PRD and not actually be representative of what's contained in this class